src/stm32/stm32h7.c

```diff
@@ -210,6 +215,10 @@ clock_setup(void)
 static void
 usb_reboot_for_dfu_bootloader(void)
 {
+#ifdef STM32H723xx
+    irq_disable();
+    *(uint64_t*)USB_BOOT_FLAG_ADDR = USB_BOOT_FLAG;
+#endif
     irq_disable();
     *(uint64_t*)USB_BOOT_FLAG_ADDR = USB_BOOT_FLAG;
```

Comment:
Just wild guess: Do this variant work?
*(volatile uint64_t*)USB_BOOT_FLAG_ADDR = USB_BOOT_FLAG;

Reply:
Tested `volatile`, it does not work, and execute `barrier()` after `*(uint64_t*)USB_BOOT_FLAG_ADDR = USB_BOOT_FLAG`, also has no effect.

Reply:
Another guess is to use SCB_CleanDCache();
Preferably with barrier () or volatile.

Reply:
Thank you very much for your help.

```
irq_disable();
*(volatile uint64_t*)USB_BOOT_FLAG_ADDR = USB_BOOT_FLAG;
barrier();
SCB_CleanDCache();
```
Just tested and still not working.